go.mod

```diff
@@ -5,18 +5,19 @@ go 1.19
 require (
 	github.com/golangci/golangci-lint v1.50.1
 	github.com/google/uuid v1.3.0
-	github.com/onsi/ginkgo/v2 v2.5.0
-	github.com/onsi/gomega v1.24.1
-	github.com/openshift/api v0.0.0-20220930192943-199dc0598fd1
+	github.com/onsi/ginkgo/v2 v2.7.0
+	github.com/onsi/gomega v1.26.0
+	github.com/openshift/api v0.0.0-20221004120407-c46852673d03
 	github.com/openshift/cluster-autoscaler-operator v0.0.1-0.20220930224803-cdc49524de94
 	github.com/openshift/library-go v0.0.0-20220930165906-a18683c20aee
 	github.com/openshift/machine-api-operator v0.2.1-0.20220922080915-f7f326323f37
 	k8s.io/api v0.25.1
 	k8s.io/apimachinery v0.25.1
-	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/client-go v0.25.1
 	k8s.io/klog v1.0.0
 	k8s.io/utils v0.0.0-20220823124924-e9cbc92d1a73
 	sigs.k8s.io/controller-runtime v0.13.0
+	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20221222054525-3c4deba74e5c
```

Comment:
Is this correct? I didn't think setup-entest was needed at this level anymore?

Reply:
Forgot to remove this. Thanks